app/javascript/packages/normalize-yaml/cli.js

```diff
@@ -24,10 +27,17 @@ const options = {
   ),
 };
 
-Promise.all(
-  files.map(async (relativePath) => {
-    const absolutePath = join(process.cwd(), relativePath);
-    const content = await readFile(absolutePath, 'utf8');
+let exitCode = 0;
+
+for await (const relativePath of files) {
```

Comment:
It just occurs to me now, but moving to top-level await has the downside of un-parallelizing the normalization (where `Promise.all` would run each async function in parallel). I might want to revert that.

Reply:
> It just occurs to me now, but moving to top-level await has the downside of un-parallelizing the normalization (where `Promise.all` would run each async function in parallel). I might want to revert that.

Updated in 1fbf107dd.